src/oc_erchef/schema/t/test_create_and_update_users.sql

```diff
@@ -0,0 +1,216 @@
+CREATE OR REPLACE FUNCTION test_zzzzcreate_and_update_users()
```

Comment:
Why "zzzz"?

Reply:
wanted that test to run last - will edit.